how_to_contribute.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to Andromeda
+
+If you want to contribute to Andromeda means that you're awesome!
+
+The following steps will help to contribute on the right way and get your PR merged easily.
+
+## Code of conduct
+
+Andromeda has adopted the [Contributor Covenant](https://www.contributor-covenant.org/) as its Code of Conduct. We expect that our contributors be aware of it. Please read the full text before start on code, it can be seen [here](./code_of_conduct.md).
+
+## Your First Pull Request
+
+Keep in mind that Andromeda is a community project. 
+Pull Requests are always welcome, but, before working on a large change, it is best to open an issue first to discuss it with the maintainers.
+This video series is recommended to understand [How to contribute to an open source project on github](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github)
+
+Thanks to [MaterialUI](https://material-ui.com/) to its incredible docs, we followed they tips on this section
```

Comment:
I think we should move this to another part of the document. Maybe the top or bottom. I fell like this is out of context here. Maybe we can reformulate this phrase, what do you think?
Many thanks to [MaterialUI](https://material-ui.com/), this document was heavily inspired by their awesome documentation.